justfile

```diff
@@ -15,7 +15,12 @@ watch:
     @echo "Watching slides"
     typst w slides.typ
 
-# Compile and Run the C code
-c:
-    @echo "Compiling and Running the C code"
-    @echo "TODO!"
\ No newline at end of file
+# Compile the C code in "code/"
```

Comment:
This docstring needs updating:

```suggestion
# Compile the C code in "solutions/"
```

Reply:
inst it right ? the solution dir contains the corrected code.

Reply:
Yes but look down below you are doing:

```just
    for FILE in $(ls ./solutions/)
    do
	    cc ./solutions/$FILE -o ./output/$FILE.out
    done
```